modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
++
+See <<Splunk Configuration Options>>.
++
+The *SSL Protocol* option applies only to the REST API data input type.
++
+[IMPORTANT]
+If you set values for the Splunk index, Splunk source, and Splunk source type when you registered your data input in Splunk, any values you specify in this window overwrites those values.
 
-|`splunkSource`
-|String
-|The source used on the events sent to Splunk.
-|Optional
-|mule
+. Click *Apply* to save your changes.
 
-|`splunkSourceType`
-|String
-|The sourcetype used on the events sent to Splunk.
-|Optional
-|mule
+=== Splunk Configuration Options
 
-|`dateFormatPattern`
-|String
-|Date format used to format the timestamp.
-|Optional
-|yyyy-MM-dd'T'HH:mm:ssSZ
+Splunk integration includes the following configuration options:
 
+[cols="20,30,20"]
 |===
+|Field|Description|Default Value
 
-.Splunk Internal Handler Minimum Configuration Example
-[source,yaml]
-....
----
-  mule.agent.gw.http.handler.splunk:
-    host: 192.168.61.131
-    user: admin
-    pass: test
-....
-
-=== Rest API
+|Username
+(Required)
+|Username to connect to Splunk
+|
 
-This feature requires the Runtime Manager agent 1.2.0 version or later.
+|Password 
+(Required)
+|Password associated with the Splunk username 
+|
 
-. Select the server whose information you want to send out.
-. In the menu on the right, click *Manage Server* to access the server's settings.
-+
-image::sending-data-from-arm-to-external-monitoring-software-manage-server.png[]
+|host
+(Required)
+|IP address or hostname of the server where Splunk is running
+|
 
-. Select the *Plugins* tab:
-+
-image::sending-data-from-arm-to-external-monitoring-software-plugins.png[]
+|port
+|Splunk management port
+|8089
 
-. Select the kind of information that you want to send out in the *Level* dropdown menu.
-+
-image::sending-data-from-arm-to-external-monitoring-software-level.png[]
+|protocol 
+a|Protocol for the connection to the Splunk management port
 
-. On the *Event Tracking* region, activate the *Splunk* switch. A pop-up menu opens where you provide your Splunk user and password data, as well as the host and port for the connection.
-+
-image::sending-data-from-arm-to-external-monitoring-software-splunk.png[]
+Possible values are http, https, and tcp.
+|https
 
-. Optionally, you can open the advanced menu and set up certain formatting properties of the data that will be sent out.
-+
-image::agent-to-splunk-restapi-advanced.png[splunk]
+|SSL Protocol (REST API only)
+a|SSL Security Protocol to use for the https connection
 
-=== HTTP Event Collector
+Possible values are TLSv1_2, TLSv1_1, TLSv1, and SSLv3.
+|TLSv1_2
 
-This feature require 1.3.1 agent version or later.
+|Splunk Index
+a|Splunk index name to which to send all events
 
-[NOTE]
-This feature requires the Runtime Manager agent version 1.3.1 or later.
+If the index doesn't exist, the internal handler creates it if the user has permission.
```

Comment:
This might be putting too fine a point on it:
```suggestion
If the index doesn't exist, the internal handler creates it if the permissions associated with the username are adequate.
```